docs/quickstart.rst

```diff
@@ -0,0 +1,16 @@
+ABC-Classroom Quickstart
+------------------------
+
+To set up a template directory to run abc-classroom in, run:
+
+    $ abc-quickstart
+
+This will create a directory for you that contains pre-made directories for templates and cloned files, as well as
+a sample ``config.yml`` file that can be modified to run the program for your classroom. ``abc-quickstart`` has
+two arguments that can be used to modify its functionality.
+
+1. ``--course_name course-repo-custom-name-here`` This argument will allow you to modify the name of the main
+repository ``abc-quickstart`` will create.
+2. ``-f Boolean`` This argument will allow you to override an existing output from ``abc-quickstart``. Doing this will
```

Comment:
docs should say just `-f` not `-f Boolean` (i.e. you simply add the `-f` flag, not `-f True`)